pkgs/create-neon/data/templates/ts/index.cts.hbs

```diff
@@ -0,0 +1,7 @@
+const addon = require('./load.cjs');
+
+export type Greeting = "hello node";
```

Comment:
It returns `"hello node"`. I was trying to make the example interesting enough for the library to export two things, so that it's a little clearer that `index.mts` has to re-export them all (which, as you say, is a bummer). I could also make the example something like:

```js
export type Greeting = { message: string };

export function hello(): Greeting {
  const message = addon.hello();
  return { message };
|
```

Reply:
I was looking at the sample project instead of at the code here. I guess that was modified to return `"hello"` instead?

Reply:
Hm, shouldn't be: https://github.com/dherman/create-neon-manual-test-project/blob/main/src/lib.rs#L4

Maybe you mixed it up with the `cx.export_function("hello", hello)?;` line?